ShapeDungeon/Data/AppDbContext.cs

```diff
@@ -16,6 +16,43 @@ protected override void OnModelCreating(ModelBuilder builder)
         {
             builder.Entity<EnemyRoom>()
                 .HasKey(x => new { x.EnemyId, x.RoomId });
+
+            // TODO: Move to a separate class to keep this method small.
```

Comment:
TODO: DB will be seeded with a whole scenario in which the user can play around, see how the app works, and then be given access to create their own scenarios.